agrirouter-api-dotnet-standard-test/Service/Onboard/OnboardServiceTest.cs

```diff
@@ -11,14 +11,34 @@
 namespace Agrirouter.Api.Test.Service.Onboard
 {
     /// <summary>
-    /// Functional tests.
+    ///     Functional tests.
     /// </summary>
     [Collection("Integrationtest")]
     public class OnboardServiceTest : AbstractIntegrationTest
     {
         private static readonly UtcDataService UtcDataService = new UtcDataService();
         private static readonly HttpClient HttpClient = HttpClientFactory.HttpClient();
 
+        [Fact]
+        public void GivenInvalidRequestTokenWhenOnboardingThenThereShouldBeAValidResponse()
```

Comment:
Wouldn't GivenInvalidRequestTokenWhenOnboardingThenThereShouldBeAnException() be more accurate?